### DISCUSSION TITLE
Highlight certain keywords

### DISCUSSION THREAD

**Discussion by umsheikh** [gonzalezreal/swift-markdown-ui#196] [Q&A] [answered]
Hi! Thanks for this amazing library. I am trying to implement a search feature and I was wondering if it's possible to apply certain styling to the searched keywords. For example, if the user searches for "This", then I want to change the text background color to yellow wherever the word "This" appears in the markdown. Appreciate any suggestions you can provide.

**Comment by gonzalezreal** [comment, marked as answer]
Hi @umsheikh,

You could achieve that by using Markdown syntax to emphasize every search term occurrence in the results (i.e., `"*this*"`) and customizing the emphasis text style in a `Markdown` view to use the background color you wish.

However, I think that using MarkdownUI for this use case is a bit of a killer. You can achieve the same by setting the background attribute in an `AttributedString` and rendering it with a `Text` view.

**Reply by yousefNowPay** [reply]
Hi @gonzalezreal  I tried what u have mentioned but "*this*" not always working is there any syntax to highlight a text inside of block of code ?
